showcase/openfe_showcase.ipynb

```diff
@@ -2,12 +2,11 @@
  "cells": [
```

Comment:
<p>The subheadings here are rather small, could we bump them up a little bit in title order?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>